populate_course_run_reporting_type_and_announcement.py

```diff
@@ -0,0 +1,80 @@
+import argparse
```

Comment:
I would prefer we do this correctly now to avoid kicking the can down the road. Please work with @schenedx to make that happen.